tst/schedulers/test_schedulers_api.py

```diff
@@ -90,6 +93,17 @@
         do_minimize=False,
         random_seed=random_seed,
     ),
+    # Multi-objective methods
+    SingleFidelityScheduler(
+        config_space,
+        searcher=ExpectedHyperVolumeImprovement(
+            config_space=config_space,
+            random_seed=random_seed,
+        ),
+        metrics=[metric1, metric2],
+        do_minimize=False,
```

Comment:
Not for this PR, but we should probably think on an easy way to specify mode per metrics in the case of multi-objective models.